### PR TITLE
feat: cherry-pick suggestions based on version info in mender-docs

### PR DESCRIPTION
**Pull request by snusifer**
Changelog: None
Ticket: QA-381
Signed-off-by: Mikael Torp-Holte <mikael.torp-holte@northern.tech>

### CODE REVIEW

File: cherrypicks.go [hunk 1]
```diff
@@ -132,12 +185,6 @@ func suggestCherryPicks(
 		return nil
 	}
 
-	// suggest cherry-picking with a comment
```
Comment (oleorhagen) [inline cherrypicks.go:135]: Q: Is it not better to keep this close to where it's used still?

File: cherrypicks.go [hunk 2]
```diff
@@ -101,10 +144,20 @@ func suggestCherryPicks(
...
+	tmplString := `
+Hello :smile_cat: This PR contains changelog entries. Please, verify the need of backporting it to the following release branches:
+{{.ReleaseBranches}}
+`
 	// get list of release versions
-	versions, err := getLatestIntegrationRelease(3, conf)
-	if err != nil {
-		return err
+	versions, err := getLatestReleaseFromApi(versionsUrl)
+	if err != nil || len(versions) == 0 {
```
[the remark this answers is not in the document]
Comment (snusifer) [reply cherrypicks.go:155]: Yeah, no I agree. I missed that the function will not return `err` if the response payload doesn't contain the `lts` field.

File: cherrypicks.go [hunk 3]
```diff
@@ -101,10 +144,20 @@ func suggestCherryPicks(
...
+	// nolint:lll
+	tmplString := `
+Hello :smile_cat: This PR contains changelog entries. Please, verify the need of backporting it to the following release branches:
+{{.ReleaseBranches}}
+`
 	// get list of release versions
-	versions, err := getLatestIntegrationRelease(3, conf)
-	if err != nil {
-		return err
+	versions, err := getLatestReleaseFromApi(versionsUrl)
```
[the remark this answers is not in the document]
Comment (snusifer) [reply cherrypicks.go:158]: moved it to cherrypicks.go

File: cherrypicks.go [hunk 4]
```diff
@@ -133,11 +190,6 @@ func suggestCherryPicks(
 	}
 
 	// suggest cherry-picking with a comment
-	// nolint:lll
```
Comment (oleorhagen) [inline cherrypicks.go:136]: Still moved :)

Comment (snusifer) [reply cherrypicks.go:136]: this is to disable the line length linter for the line I had to move in the first place :))

Comment (oleorhagen) [reply cherrypicks.go:136]: I see, it is because you are adding to the string the `apiWarning`.

Never mind. I missed it

**Review by oleorhagen — APPROVED** [review]
Nice :)

**Pull request merged**